src/test/java/io/arlas/server/ogc/wfs/WFSServiceIT.java

```diff
@@ -92,14 +95,19 @@ public void handleHeaderFilter(ValidatableResponse then) throws Exception {
     public void handleNoHeaderFilter(ValidatableResponse then) throws Exception {
         then.statusCode(200)
                 .body("wfs:FeatureCollection.@numberReturned", equalTo("595"))
-                .body("wfs:FeatureCollection.member[1].geodata.params_job", equalTo("Dancer"))
+                .body("wfs:FeatureCollection.member[1].geodata.params_job", isOneOf(DataSetTool.jobs))
                 .body("wfs:FeatureCollection.member[1].geodata.params_country.size()", equalTo(0))
                 .body("wfs:FeatureCollection.member[1].geodata.params_city.size()", equalTo(0));
     }
 
     public void handleDescribeFeature(ValidatableResponse then) throws Exception {
-        then.statusCode(200)
-                .body("xs:schema.complexType.complexContent.extension.sequence.element.size()", equalTo(7));
+        if(!DataSetTool.ALIASED_COLLECTION) {
+            then.statusCode(200)
+                    .body("xs:schema.complexType.complexContent.extension.sequence.element.size()", equalTo(7));
+        } else {
+            then.statusCode(200)
+                    .body("xs:schema.complexType.complexContent.extension.sequence.element.size()", equalTo(9));
```

Comment:
@mbarbet Could you have a closer look on this? I don't understand why it may be 7... and why it turns to 9 with only  one additional field in the mapping. I am not sure of what these xml elements represent.